multimedia/templates/multimedia/book_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends "multimedia/book_base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+{{ block.super }}
```

Comment:
If you need to get the content of the block from the parent template, the {{ block.super }} variable will do the trick. This is useful if you want to add to the contents of a parent block instead of completely overriding it.

https://docs.djangoproject.com/en/1.7/topics/templates/#template-inheritance